sass/directives/_carousels.scss

```diff
@@ -249,3 +249,39 @@
     margin-bottom: 0;
   }
 }
+
+@mixin logos-carousel {
+  padding-bottom: $base-padding;
+
+  .slick-track {
+    display: flex;
+    align-items: center;
+
+    img {
+      margin: auto;
+      max-width: 10em;
```

Comment:
both logo width and height should be parameters on the mixin. That makes the mixin more reusable, even if you add your current values as defaults